include/seqan3/range/container/dynamic_bitset.hpp

```diff
@@ -1286,7 +1286,7 @@ class dynamic_bitset
     template <std::forward_iterator begin_it_type, typename end_it_type>
     //!\cond
         requires std::sentinel_for<end_it_type, begin_it_type> &&
-                 std::constructible_from<value_type, /*ranges::iter_reference_t*/reference_t<begin_it_type>>
+                 std::constructible_from<value_type, ranges::iter_reference_t<begin_it_type>>
```

Comment:
```suggestion
                 std::constructible_from<value_type, std::iter_reference_t<begin_it_type>>
```

Reply:
Oh, interesting that these did not break things when I deleted the seqan3::reference struct as part of flushing out all the occurrences.. Sorry to have missed it!